trace/google/cloud/trace/v1/_gax.py

```diff
@@ -0,0 +1,206 @@
+# Copyright 2017 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""GAX Wrapper for interacting with the Stackdriver Trace API."""
+
+from google.api_core import page_iterator
+from google.cloud.trace_v1.gapic import trace_service_client
+from google.cloud.trace_v1.proto import trace_pb2
+from google.gax import CallOptions
+from google.gax import INITIAL_PAGE
+from google.protobuf.json_format import MessageToDict
+from google.protobuf.json_format import ParseDict
+
+
+class _TraceAPI(object):
+    """Wrapper to help mapping trace-related APIs.
+
+    See
+    https://cloud.google.com/trace/docs/reference/v1/rpc/google.devtools.
+    cloudtrace.v1
+
+    :type gax_api:
+        :class:`~google.cloud.trace_v1.gapic.trace_service_client.
+               TraceServiceClient`
+    :param gax_api: API object used to make GAX requests.
+
+    :type client: :class:`~google.cloud.trace.client.Client`
+    :param client: The client that owns this API object.
+    """
+    def __init__(self, gax_api, client):
+        self._gax_api = gax_api
+        self.client = client
+
+    def patch_traces(self, project_id, traces, options=None):
+        """Sends new traces to Stackdriver Trace or updates existing traces.
+
+        :type project_id: str
```

Comment:
Just to check, is there any reason we need to leave these with the old-style docstrings?  I.e. we have napoleon above like:

```
Args:
    start_time (:class:`~datetime.datetime`):
               The start time of the span. 
```

vs the `:type`, `:param` style here.

If there isn't a reason to keep them different, then we should file an issue against ourselves to go clean everything up to a single consistent style (some day).

Reply:
@duggelz We've been "fine with both" for some time now. However we've privately discussed being OK with badgering Googlers to exclusively use Google-style docstrings.

Reply:
Changed to use napoleon style.